ducttape/data_sources/lexia.py

```diff
@@ -66,7 +66,7 @@ def _login(self):
         """ Logs into the provided Lexia instance.
         """
         login_url = self.uri_scheme + 'auth.mylexia.com/mylexiaLogin'
-        self.log.debug('Logging into Lexia at: {}'.format(login_url))
+        self.log.info('Logging into Lexia at: {}'.format(login_url))
```

Comment:
When I was first writing this, I wasn't sure that a user would want all of this info in every task log. I think I agree that I probably would at this point in the case of a task failure.